build_tools/benchmarks/common/benchmark_config_test.py

```diff
@@ -18,12 +18,16 @@ class BenchmarkConfigTest(unittest.TestCase):
 
   def setUp(self):
     self.build_dir = tempfile.TemporaryDirectory()
+    self.build_dir_name = os.path.realpath(self.build_dir.name)
```

Comment:
tempfile doesn't give us a usable path here?! Can you elaborate on what the issue is?

Reply:
On macs the returned path is symlinked. Because we already ask for the `os.realpath` in the BenchmarkConfig it receives the symlinked folder instead of the symlink. This is more of a quirk from BenchmarkConfig specifically grabbing the realpath for its behavior.